client/src/components/LinkDisplay/index.js

```diff
@@ -45,11 +46,22 @@ const LinkDisplay = ({ content }) => {
           </button>
         </div>
       </div>
+      <div
+        className={`${styles.pinDisplay} ${darkMode ? styles.darkOpenLink : styles.lightOpenLink}`}
+      >
+        <span className={styles.pinDisplayMsg}>Anyone with the PIN also can join the chat</span>
+        <PinDisplay content={content.pin} />
+        <span className={styles.pinValidMsg}>PIN is valid for 30 minutes</span>
+      </div>
       <div
         className={`${styles.openLink}
       ${darkMode ? styles.darkOpenLink : styles.lightOpenLink}`}
       >
-        <a href={content} target="_blank" rel="noopener noreferrer">
+        <a
+          href={`${content.absoluteLink}?pin=${content.pin}`}
```

Comment:
Adding PIN in url parameter - it's a security vulnerability. 
Please add a **To Do** comment that "this needs to be changed - make an POST endpoint to fetch the PIN of an url"